src/main/resources/org/jenkinsci/plugins/tuleap_credentials/TuleapAccessTokenImpl/config.jelly

```diff
@@ -0,0 +1,9 @@
+<?xml version="1.0" encoding="utf-8"?>
+<?jelly escape-by-default='true'?>
+
+<j:jelly xmlns:j="jelly:core" xmlns:f="/lib/form" xmlns:st="jelly:stapler">
+    <f:entry title="${%Token}" field="token">
```

Comment:
It seems there is no .properties associated to this file while you need it since there is ${%Token}